test/helpers/describe-contract.ts

```diff
@@ -21,6 +21,61 @@ export interface TestContext extends DeployedContracts {
 chai.use(dirtyChai);
 chai.use(solidity);
 
+export function mainnetForkTest(
+  name: string,
+  test: () => Promise<void>,
+) {
+  if (config.FORK_MAINNET) {
+    it(`Running mainnet fork test: ${name}`, test);
+  } else {
+    it.skip(`Skipping mainnet fork test: ${name}`, test);
+  }
+}
+
+export function describeContractHardhat(
```

Comment:
Wasn't sure what to name this function, open to suggestions.

Reply:
Seems ok, could clarify the purpose of it with a short docstring